fe/src/main/java/org/apache/doris/alter/RollupHandler.java

```diff
@@ -406,6 +413,48 @@ public void processDropRollup(DropRollupClause alterClause, Database db, OlapTab
         LOG.info("finished drop rollup index[{}] in table[{}]", rollupIndexName, olapTable.getName());
     }
 
+    // sql: alter table base_table_name set property ("storage_format" = "v2");
```

Comment:
This grammar is confusing. I think that grammar is used to changing all base table to v2 format.
But is creating rollup actually.

Reply:
Ok, I will reserve this grammar for schema change.
I will use alter table xxx add rollup xxx grammar to realize adding segment v2 rollup